test/Confluent.Kafka.Avro.IntegrationTests/Tests/PrimitiveTypes.cs

```diff
@@ -136,8 +136,8 @@ public static void PrimitiveTypes(string bootstrapServers, string schemaRegistry
                 consumer.Assign(new List<TopicPartitionOffset> { new TopicPartitionOffset(boolTopic, 0, 0) });
                 consumer.Consume(out Message<bool, bool> message, TimeSpan.FromSeconds(10));
                 Assert.Equal(ErrorCode.NoError, message.Error.Code);
-                Assert.Equal(true, message.Key);
-                Assert.Equal(false, message.Value);
+                Assert.True(message.Key);
```

Comment:
i've made a heap of these types of changes, which were causing compiler warnings.